problems/scope/problem.md

```diff
@@ -64,7 +64,7 @@ var a = 1, b = 2, c = 3;
 Use your knowledge of the variables' `scope` and place the following code inside one of the functions in 'scope.js'
 so the output is `a: 1, b: 8,c: 6`
```

Comment:
Also this one has to be changed to `a: 1, b: 8, c: 6`. Notice a space between `,` and `c`.